Yosemite/Yosemite/Model/Orders/OrderItem+PricePreDiscount.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+import WooFoundation
+
+public extension OrderItem {
+    /// Calculates a single line item price before discount and taxes
+    /// This value correctly reflects a single line item price within order
+    ///
+    /// OrderItem subtotal - excludes discounts and depends on item quantity
+    /// OrderItem total - includes discounts and depends on item quantity
+    /// OrderItem price - includes discounts
+    /// Product price - always static and doesn't take into
+    /// account whether taxes are included or excluded from product
+    ///
+    /// - Parameter orderItem
+    ///
+    var pricePreDiscount: NSDecimalNumber {
```

Comment:
The name of `pricePreDiscount` seems to imply that there's always a discount to be applied after, which may not be the case. Also makes no reference to being affected by tax. What about using a name like `netPrice`, `basePrice`, or similar?

Reply:
Good point. The only reason I named it this way was because Android used this naming https://github.com/woocommerce/woocommerce-android/blob/1e2829c413233e7a1a8602137ea8b2858b377386/WooCommerce/src/main/kotlin/com/woocommerce/android/model/Order.kt#L126

Android chose this naming because `orderItem.price` is a single item price that includes discount, `orderItem.pricePreDiscount` is same as `orderItem.price` minus discount.